packages/react/src/components/DashboardEditor/DashboardEditor.jsx

```diff
@@ -372,6 +373,14 @@ const DashboardEditor = ({
   // eslint-disable-next-line react/prop-types
   onFetchDynamicDemoHotspots, // needed for the HotspotEditorModal, see the proptypes for more details
 }) => {
+  React.useEffect(() => {
```

Comment:
should we swap all of the components to show their warnings like this? It does make more sense to only display it once